grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -518,6 +524,69 @@ class OrderController {
         }
     }
 
+    def orderItemFormDialog = {
+        log.info "params " + params
```

Comment:
Hmm... I am thinking if this loggers are required here or should i mark for removal those as consol.logs in js?

Reply:
Yeah I meant to clean that up. It was used for debugging. Will do the same for console.log